examples/plotting/file/choropleth.py

```diff
@@ -1,3 +1,13 @@
+'''This example shows an unemployment map by county of the United States of America in 2009.
+It demonstrates drawing shapes using polygonal data, as well as using color scales to indicate different integer ranges
+
+.. bokeh-example-metadata::
+    :sampledata: us_counties, us_states, unemployment
+    :apis: bokeh.plotting.figure.patches, bokeh.plotting.figure.grid.grid_line_color bokeh.palettes.Viridis6, bokeh.plotting.show
```

Comment:
```suggestion
    :apis: bokeh.plotting.figure.patches, bokeh.palettes.Viridis6, bokeh.plotting.show
```
`grid_line_color` is actually a property of the `bokeh.models.Grid` object, but I think we can just omit it since that's a bit finer grained than we have been targeting with the APIs field.